server/util/csv_parse.py

```diff
@@ -258,14 +261,37 @@ def reject_duplicate_values(
         yield row
 
 
+TOTAL_REGEX = re.compile(r"(^|[^a-zA-Z])(sub)?totals?($|[^a-zA-Z])", re.IGNORECASE)
+
+
 def reject_total_rows(csv: CSVDictIterator) -> CSVDictIterator:
     for r, row in enumerate(csv):  # pylint: disable=invalid-name
         for value in row.values():
-            if value.lower() in ["total", "totals", "total ballots", "county totals"]:
-                raise CSVParseError(f"Remove total row (row {r+2})")
+            if TOTAL_REGEX.search(value):
+                raise CSVParseError(
+                    f"It looks like you might have a total row (row {r+2})."
```

Comment:
Should it include the name of the row?

Reply:
That would require some knowledge about which column is the "name" of the row. We could decide that, but I don't know that it's necessary

Reply:
I think I was thinking "column" when I wrote this. Nevermind!